modules/swagger-codegen-maven-plugin/examples/java-client.xml

```diff
@@ -101,15 +96,16 @@
             <groupId>junit</groupId>
             <artifactId>junit</artifactId>
             <version>${junit-version}</version>
-            <scope>test</scope>
+            <!--<scope>test</scope>-->
         </dependency>
     </dependencies>
     <properties>
         <swagger-annotations-version>1.5.0</swagger-annotations-version>
-        <jersey-version>2.12</jersey-version>
-        <jackson-version>2.4.2</jackson-version>
-        <jodatime-version>2.3</jodatime-version>
+        <swagger-core-version>1.5.8</swagger-core-version>
```

Comment:
I used the output from https://generator.swagger.io/#!/clients/generateClient in order to get the version numbers. That generator API returns 1.5.8 so I copied it in order to stay on par with what was already being done.

I can update it if you think there are no breaking changes between versions.